deploy/pivot_role/pivotRoleCDK/dataall_base_infra.py

```diff
@@ -0,0 +1,827 @@
+                                         actions=[
+                                             "secretsmanager:DescribeSecret",
+                                             "secretsmanager:GetSecretValue"
+                                         ],
+                                         resources=[
+                                             f"arn:aws:secretsmanager:*:{self.account}:secret:{env_resource_prefix}*",
+                                             f"arn:aws:secretsmanager:*:{self.account}:secret:dataall*"
+                                         ]
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="SecretsManagerList", effect=iam.Effect.ALLOW,
+                                         actions=["secretsmanager:ListSecrets"],
+                                         resources=["*"]
+                                     ),
+                                     # IAM
+                                     iam.PolicyStatement(
+                                         sid="IAMList", effect=iam.Effect.ALLOW,
+                                         actions=["iam:ListRoles"],
+                                         resources=["*"]
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="IAMRolePolicy", effect=iam.Effect.ALLOW,
+                                         actions=[
+                                             "iam:PutRolePolicy",
+                                             "iam:DeleteRolePolicy"
+                                         ],
+                                         resources=["*"]
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="IAMPassRole", effect=iam.Effect.ALLOW,
+                                         actions=[
+                                             "iam:Get*",
+                                             "iam:PassRole"
+                                         ],
+                                         resources=[
+                                             f"arn:aws:iam::{self.account}:role/{env_resource_prefix}*",
+                                             f"arn:aws:iam::{self.account}:role/{role_name}",
+                                             f"arn:aws:iam::{self.account}:role/cdk-*"
+                                         ],
+                                     ),
+                                     # STS
+                                     iam.PolicyStatement(
+                                         sid="STS", effect=iam.Effect.ALLOW,
+                                         actions=["sts:AssumeRole"],
+                                         resources=[f"arn:aws:iam::{self.account}:role/{env_resource_prefix}*"]
+                                     ),
```

Comment:
Allow sts:AssumeRole for `'arn:aws:iam::${AWS::AccountId}:role/ddk-*'` for data.all DDK Pipelines